spec/features/user_views_an_item_type_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
```

Comment:
No you are correct...although that leaves some other holes in some other apps...Carry on.